### DISCUSSION TITLE
Bring some Christmas Spirit to your Public Page (Christmas Lights & Snowflakes)

### DISCUSSION THREAD

**Discussion by gAlleb** [AzuraCast/AzuraCast#6820] [Show and tell]
A lit bit of Xmas for your Azuracast Station public page (very dirty but quick `flashing Christmas Lights` ;) ). 

[Gist](https://gist.github.com/gAlleb/8df10a0c40f461909f2a060abf945e1a)

[Instructions](https://gist.github.com/gAlleb/8df10a0c40f461909f2a060abf945e1a?permalink_comment_id=4805745#gistcomment-4805745)

15 seconds – 1 minute (depending on your copy/paste speed :)) and done! :) Result:
<img width="500" alt="Снимок экрана 2023-12-24 в 18 54 07" src="https://github.com/AzuraCast/AzuraCast/assets/144447266/4e973f0c-f3d2-4373-87cb-9d0d609b1ce5">

Wishing you a Merry Christmas and a Happy New Year!

Thanks Azuracast developers and collaborators ;) for great and ever-evolving product!

**Comment by Moonbase59** [comment]
Great idea. Merry Christmas to all!

**Comment by gAlleb** [comment]
Another bit of Xmas for your Azuracast Station public page — Snoflakes! 

Snoflakes are falling down. Real Snowflakes omg :)

[Gist](https://gist.github.com/gAlleb/44f4d615f22e11a6cfb78af1d9969078)

[Instructions](https://gist.github.com/gAlleb/44f4d615f22e11a6cfb78af1d9969078?permalink_comment_id=4806676#gistcomment-4806676)

15 seconds – 1 minute (depending on your copy/paste speed :)) and done! :) Result:
<img width="500" alt="Снимок экрана 2023-12-25 в 14 36 29" src="https://github.com/AzuraCast/AzuraCast/assets/144447266/295fb731-3f6e-4f53-9f12-e4114cd3c07a">
